rdrf/rdrf/helpers/cache_utils.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from django.conf import settings
+from django.core.cache import caches
+from ccg_django_utils.conf import EnvConfig
+
+logger = logging.getLogger(__name__)
+env = EnvConfig()
+
+
+def use_cache(function):
+    def wrapper(*args, **kwargs):
+
+        if settings.CACHE_DISABLED or env.get("CACHE_DISABLED", False):
+            return function(*args, **kwargs)
+
+        key = f"{function.__name__}"
```

Comment:
this is a static key so won't work  when arguments are different

Reply:
best to remove this if not being used

Reply:
removed